src/ucp/core/ucp_ep.c

```diff
@@ -2507,13 +2536,23 @@ void ucp_ep_print_info(ucp_ep_h ep, FILE *stream)
     ucp_lane_index_t wireup_msg_lane;
     ucs_string_buffer_t strb;
     uct_ep_h wireup_ep;
+    ucs_memory_type_t mem_type;
 
     UCP_WORKER_THREAD_CS_ENTER_CONDITIONAL(worker);
 
-    fprintf(stream, "#\n");
-    fprintf(stream, "# UCP endpoint\n");
-    fprintf(stream, "#\n");
-    fprintf(stream, "#               peer: %s\n", ucp_ep_peer_name(ep));
+    mem_type = ucp_worker_lookup_ep_mem_type(worker, ep);
+    if (mem_type == UCS_MEMORY_TYPE_LAST) {
+        fprintf(stream, "#\n");
+        fprintf(stream, "# UCP endpoint\n");
+        fprintf(stream, "#\n");
+        fprintf(stream, "#               peer: %s\n", ucp_ep_peer_name(ep));
+    } else {
+        fprintf(stream, "#\n");
+        fprintf(stream, "# UCP endpoint for %s (\"%s\" memory type)\n",
+                ucs_memory_type_descs[mem_type],
+                ucs_memory_type_names[mem_type]);
```

Comment:
would be enough to print only one of {desc,name}

Reply:
done